src/i18n/i18nFactory.ts

```diff
@@ -13,50 +13,50 @@ import { zhTranslation } from './resources/zh'
 // @ts-ignore
 const createInstance = i18next.createInstance || i18next.default.createInstance
 
-const translationsFiles: { [langCode: string]: any } = {
-  en: enTranslation,
-  es: esTranslation,
-  fr: frTranslation,
-  ru: ruTranslation,
-  ar: arTranslation,
-  zh: zhTranslation,
+const translationsFiles = {
+  [Lang.en]: enTranslation,
+  [Lang.es]: esTranslation,
+  [Lang.fr]: frTranslation,
+  [Lang.ru]: ruTranslation,
+  [Lang.ar]: arTranslation,
+  [Lang.zh]: zhTranslation,
 }
 
-export const createParams = (lang: string) => ({
-  fallbackLng: 'en',
+export const createParams = (lang: Lang) => ({
+  fallbackLng: Lang.en,
   debug: false,
 
   // react i18next special options (optional)
   react: {
     wait: false, // set to true if you like to wait for loaded in every translated hoc
-    nsMode: 'default', // set it to fallback to let passed namespaces to translated hoc act as fallbacks
+    nsMode: 'default' as 'default' | 'fallback', // set it to fallback to let passed namespaces to translated hoc act as fallbacks
```

Comment:
This fixes a ts error we get at `src/client/i18n.ts` : `Type 'string' is not assignable to type '"default" | "fallback"'.`